lib/pengine/clone.c

```diff
@@ -1319,6 +1319,27 @@ pe__set_clone_flag(pe_resource_t *clone, enum pe__clone_flags flag)
     return pcmk_rc_ok;
 }
 
+/*!
+ * \internal
+ * \brief Check whether a clone flag is set
+ *
+ * \param[in] group  Clone resource to check
+ * \param[in] flags  Flag or flags to check
+ *
+ * \return \c true if all \p flags are set for \p clone, otherwise \c false
+ */
+bool
+pe__clone_flag_is_set(const pe_resource_t *clone, uint32_t flags)
+{
+    clone_variant_data_t *clone_data = NULL;
+
+    CRM_ASSERT((clone != NULL) && (clone->variant == pe_clone)
+               && (clone->variant_opaque != NULL));
+    get_clone_variant_data(clone_data, clone);
```

Comment:
get_clone_variant_data() does the first two asserts itself

Reply:
good catch